Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -235,7 +256,7 @@ class InstructionPresenter {
             genericRouteShield.cornerRadius = appearance.cornerRadius
             
             guard let image = takeSnapshot(on: genericRouteShield) else { return nil }
-            imageRepository.storeImage(image, forKey: key, toDisk: false)
+            spriteRepository.legacyCache.store(image, forKey: key, toDisk: false, completion: nil)
```

Comment:
Generic shield and exit views will be stored in the `legacyCache`, as the same as the legacy shield icons. When the style change or the `traitCollection` change, they will be removed and re-generated and displayed in the label, to avoid the style change delay.